server/src/main/java/org/eclipse/lsp/cobol/core/model/variables/RenameItem.java

```diff
@@ -37,8 +37,8 @@
 @EqualsAndHashCode(callSuper = true)
 public class RenameItem extends StructuredVariable {
 
-  public RenameItem(@NonNull String name, @NonNull Locality definition) {
-    super(LEVEL_66, name, definition, null);
+  public RenameItem(@NonNull String name, Boolean global, @NonNull Locality definition) {
```

Comment:
This one is tricky, it may depend on the renamed variable visibility

Reply:
Great catch! Now I set it properly via `variable.precedingStructure.isGlobal()`.

Reply:
Resolve it the other way.